packages/cli/commands/project/download.js

```diff
@@ -34,10 +34,11 @@ exports.handler = async options => {
   await loadAndValidateOptions(options);
 
   const { projectConfig } = await getProjectConfig();
-  const platformVersion =
-    projectConfig && projectConfig.platformVersion
-      ? projectConfig.platformVersion
-      : '';
+
+  if (projectConfig) {
+    logger.warn(i18n(`${i18nKey}.warnings.cannotDownloadWithinProject`));
+    process.exit(EXIT_CODES.WARNING);
```

Comment:
The `WARNING` exit status is reserved for things like validation issues. This is less of a warning and more of an error. I think we should use error status here because this is a scenario where the command cannot possibly run.